src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml

```diff
@@ -16,361 +16,361 @@
-        @Html.Gds().ValidationMessageFor(m => m.SelectedComplianceYear)
-        <span class="text-margin">
-            @Html.Gds().DropDownListFor(m => m.SelectedComplianceYear, new SelectList(Model.ComplianceYearList), new { @class = "govuk-!-width-one-quarter", autocomplete = "false" })
-        </span>
-    </div>
+<div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedComplianceYear)">
+    <span class="font-bold govuk-body">@Html.LabelFor(m => m.SelectedComplianceYear):</span>
+    @Html.Gds().ValidationMessageFor(m => m.SelectedComplianceYear)
+    <span class="text-margin">
+        @Html.Gds().DropDownListFor(m => m.SelectedComplianceYear, new SelectList(Model.ComplianceYearList), new { @class = "govuk-!-width-one-quarter", autocomplete = "false" })
+    </span>
+</div>
 
-    <div class="govuk-tabs" data-module="govuk-tabs">
-        <h2 class="govuk-tabs__title">
-            Contents
-        </h2>
+<div class="govuk-tabs" data-module="govuk-tabs">
+    <h2 class="govuk-tabs__title">
+        Contents
+    </h2>
 
-        <ul class="govuk-tabs__list">
-            <li class="govuk-tabs__list-item">
-                <a class="govuk-tabs__tab" href="#aatfDetails">
-                    @Model.FacilityType.ToDisplayString() details
-                </a>
-            </li>
-            <li class="govuk-tabs__list-item">
-                <a class="govuk-tabs__tab" href="#data">
-                    Data
-                </a>
-            </li>
-            <li class="govuk-tabs__list-item">
-                <a class="govuk-tabs__tab" href="#organisationDetails">
-                    Organisation details
-                </a>
-            </li>
-            <li class="govuk-tabs__list-item">
-                <a class="govuk-tabs__tab" href="#contactDetails">
-                    Contact details
-                </a>
-            </li>
-        </ul>
+    <ul class="govuk-tabs__list">
+        <li class="govuk-tabs__list-item">
+            <a class="govuk-tabs__tab" href="#aatfDetails">
+                @Model.FacilityType.ToDisplayString() details
+            </a>
+        </li>
+        <li class="govuk-tabs__list-item">
+            <a class="govuk-tabs__tab" href="#data">
+                Data
+            </a>
+        </li>
+        <li class="govuk-tabs__list-item">
+            <a class="govuk-tabs__tab" href="#organisationDetails">
+                Organisation details
+            </a>
+        </li>
+        <li class="govuk-tabs__list-item">
+            <a class="govuk-tabs__tab" href="#contactDetails">
+                Contact details
+            </a>
+        </li>
+    </ul>
 
-        <section class="govuk-tabs__panel" id="aatfDetails">
-            <header>
-               <h1 class="govuk-heading-l">
+    <div class="govuk-tabs__panel" id="aatfDetails" tabindex="0">
+        <h1 class="sr-only" tabindex="0">AATF Details</h1>
```

Comment:
Change this to @Model.FacilityType.ToDisplayString()

Reply:
Updated as per the suggestion :)